documentation/library_guide/overview.rst

```diff
@@ -78,12 +78,20 @@ Known Limitations
   in the processed data sequence: ``std::iterator_traits<IteratorType>::value_type``.
 * ``exclusive_scan`` and ``transform_exclusive_scan`` algorithms may provide wrong results with
   vector execution policies when building a program with GCC 10 and using ``-O0`` option.
-* The use of oneDPL together with the GNU C++ standard library (libstdc++) version 9 or 10 may lead to
+* The use of |onedpl_short| together with the GNU C++ standard library (libstdc++) version 9 or 10 may lead to
   compilation errors (caused by oneTBB API changes). 
-  To overcome these issues, include oneDPL header files before the standard C++ header files,
+  To overcome these issues, include |onedpl_short| header files before the standard C++ header files,
   or disable parallel algorithms support in the standard library. 
   For more information, please see `Intel® oneAPI Threading Building Blocks (oneTBB) Release Notes`_.
-* The ``using namespace oneapi;`` directive in a oneDPL program code may result in compilation errors
+* A compilation error occurs if oneTBB is not in your environment when you use |onedpl_short| with a serial or OpenMP backend,
+  libstdc++ version 9, and C++17 or C++20. The error occurs because libstdc++ version 9 does not check for oneTBB availability.
+  To remove the error:
+
+  * Disable support for Parallel STL algorithms by setting the macro, ``PSTL_USE_PARALLEL_POLICIES``, to zero before
+    including the first standard header file in each translation unit.
+  * Include |onedpl_short| headers before you include other headers.
+
```

Comment:
Looking at this and the previous note, I think these should be combined. The issues are very similar, both are not caused by oneDPL, and the mitigation is exactly the same. I'd propose to add the following to the previous note (before the mitigation description):
"Also, libstdc++ version 9 requires TBB version 2020 for the ``<execution>`` header file, which may result in compilation errors when TBB is not found in the environment even if its use in |onedpl_short| is switched off."

Reply:
I partially agree with you. 
My suggestion:
> Also, libstdc++ version 9 requires TBB version 2020 for the <execution> header file, which may result in compilation errors when we use C++17 or C++20 and TBB is not found in the environment even if its use in |onedpl_short| is switched off.

Reply:
My quick edit:
Using libstdc++ version 9 requires TBB version 2020 for the header file. This may result in compilation errors when using C++17 or C++20 and TBB is not found in the environment, even if its use in |onedpl_short| is switched off.

Reply:
I agree with your wording.

> Using libstdc++ version 9 requires TBB version 2020 for the `<execution>` header file. This may result in compilation errors when using C++17 or C++20 and TBB is not found in the environment, even if its use in |onedpl_short| is switched off.